srcipt/localStorage.js

```diff
@@ -0,0 +1,30 @@
+const contactForm = document.querySelector('#contact form');
+const { fullName, message, email } = contactForm.elements;
+
+// storing the form data in local storage
+contactForm.addEventListener('change', () => {
+  const obj = {};
+
+  if (fullName.value.trim() !== '') {
+    obj.userName = fullName.value.trim();
+  }
+
+  if (email.value.trim() !== '') {
+    obj.userEmail = email.value.trim();
+  }
+
+  if (message.value.trim() !== '') {
+    obj.userMessage = message.value.trim();
+  }
+
+  localStorage.setItem('formData', JSON.stringify(obj));
+});
+
+// set the localStorage data to the form
+
+const formObj = JSON.parse(localStorage.getItem('formData'));
+const { userName, userEmail, userMessage } = formObj;
+
+fullName.value = userName || '';
+email.value = userEmail || '';
+message.value = userMessage || '';
```

Comment:
- [ ] Nice code organization you have here I must say 👏. However, after filling the form with my name, email, and text and re-loads the page, the information I entered did not persist to the page. Remember that your code should check if there are any saved data in the local storage and if so the input fields are pre-filled with the data. Consider adding an event listener to the `window` that listens to page load and then call your local storage to populate the input fields.

Check out this [link](https://www.w3schools.com/js/js_htmldom_eventlistener.asp) to see how to add an event listener to the window